src/components/10-atoms/spinner/README.md

```diff
@@ -0,0 +1,30 @@
+# Spinner
+
+Adds a loading spinner
```

Comment:
Well, this does not need an explanation. The explanation should rather be, what that component is and what it can be used for.